src/components/user-list-item/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { formatDistanceStrict } from 'date-fns';
+import React from 'react';
+
+import { ChatBriefViewResponse } from '@api/lib';
+import ChatAvatarSvg from '@public/icons/chat-avatar.svg';
+import GroupAvatarSvg from '@public/icons/group-avatar.svg';
+
+const userId = 'user001'; //temp value: logged in user id
+const unreadableMessages = 32; //temp value: number of unread messages
+
+const UserListItem: React.FC<{ item: ChatBriefViewResponse }> = ({ item }) => {
+  const { chatName, participants, lastMessage } = item;
+  const title = chatName
+    ? chatName
+    : participants
+        ?.filter((user) => user.id !== userId)
+        .map((value) => value.username)
+        .toString() || 'New Chat';
```

Comment:
Use `flatMap` instead of `map & filter`. Memoize it. Also, you don't need ternary operator